lib/tasks/teams.rake

```diff
@@ -0,0 +1,9 @@
+namespace :teams do
+  desc "Import a team and it's members from Github and install a webhook to monitor it"
```

Comment:
"Import a team and it's members" could be "Import the members of each team configured through the `github_oauth.teams` config" ?